lib/auth-client.js

```diff
@@ -157,9 +159,26 @@ AuthClient.prototype.getAuthUrl = function(options) {
     parameters.state = options.state;
   }
 
-  if (options.singleSelect !== undefined) {
-    // eslint-disable-next-line camelcase
-    parameters.single_select = options.singleSelect === true;
+  if (options.singleSelect !== undefined && options.singleSelect !== null) {
+    if (typeof options.singleSelect === 'object') {
+      const availableParams = ['vin'];
+      let singleSelectParamAdded = false;
```

Comment:
we can do the same as python to clean it up a little bit? initialize the single_select to false and then change it to true?

Reply:
yup! fixed